bin/sass.ts

```diff
@@ -12,6 +13,10 @@ try {
     compilerCommand[0],
     [...compilerCommand.slice(1), ...process.argv.slice(2)],
     {
+      // Node blocks launching .bat and .cmd without a shell due to CVE-2024-27980
+      shell: ['.bat', '.cmd'].includes(
+        path.extname(compilerCommand[0]).toLowerCase()
+      ),
```

Comment:
This is to fix local test failure: https://github.com/sass/dart-sass/actions/runs/11563285433/job/32186274921

Basically the same thing as https://github.com/sass/embedded-host-node/pull/286